fleet_vehicle_ownership/models/res_partner.py

```diff
@@ -0,0 +1,35 @@
+# Copyright 2023 RPSJR
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class ResPartner(models.Model):
+    _inherit = "res.partner"
+
+    @api.depends("vehicle_ids")
+    def _compute_vehicle_count(self):
+        for rec in self:
+            rec.vehicle_count = len(rec.vehicle_ids)
+
+    vehicle_ids = fields.One2many(
+        "fleet.vehicle",
+        "owner_id",
+        required=True,
+        help="Vehicles owned by this partner",
+    )
+    vehicle_count = fields.Integer(
+        compute=_compute_vehicle_count, string="Number of Vehicles", store=True
+    )
+
+    def action_view_vehicles(self):
+        xmlid = "fleet.fleet_vehicle_action"
+        action = self.env["ir.actions.act_window"]._for_xml_id(xmlid)
+        if self.vehicle_count > 1:
```

Comment:
I would recomend to add a default_owner_id = self.id on context

Reply:
done